### ISSUE TITLE
Start menu broken.

### ISSUE THREAD

**Issue by grimreaper1014** [Sycnex/Windows10Debloater#277]
I have used this script many times. However, this is the first time it has ever broken the start menu. It is a fresh system with a clean install of Windows 10 x64. I downloaded all the updates as normal. Then, I proceeded to run the script. I clicked remove all bloatware, disable cortana, stop edge takeover, uninstall One Drive, disable telemetry, unpin start meu, and remove bloatware reg keys. I rebooted the system. I clicked on the start menu and was presented with "start menu critical error" it gave me the option to sign out to correct the error. I clicked sign out and it continues to say that. I ran the Windows 10 start menu troubleshooter. It says that "Required applications are not installed correctly" and "Tile database is corrupt". I also ran system file checker. The problem still persists.

**Comment by Limeay** [comment]
Yep, i did the same thing and it destroy my tiles, i can't even get into the "System" portion for my display setting now cuz it wrecked havoc and took out almost everything i needed. My Start menu doesn't show my pinned applications. Everytime i press "System" in Window Settings it just closes out meaning this Windows 10 Debloater messed up my OS by removing stuff.

**Comment by Sycnex** [comment]
What versions of Windows are you all running?

**Comment by Limeay** [comment]
V2004... i can't get the windows settings -> system link to open... right when I click on it, it closes and there's a crash .dmp or something file in the crash log. I can't even upload it to github... I'm scared about doing this to my laptop at this time.... as I thought this was truly safe for my desktop. I resolved the issue for the start menu tiles. I had to reboot. Both my desktop and laptop are v2004.

It's apprently called "Immersive Control Panel", tried to rename it in windows folder and SFC /Scannow and doesn't fix it. That debloater tool literally removed a core component in there.
I tried couple of these, never tried to install the windows updates though as it'll probably not work anyways

https://windowsreport.com/problems-settings-feature-windows-10/
https://forums.tomshardware.com/threads/windows-10-settings-instantly-crashes.3371183/
https://www.dell.com/support/article/en-us/sln298531/windows-10-the-settings-app-does-not-open-or-flashes-and-closes-instantaneously?lang=en
https://softwarekeep.com/help-center/how-to-fix-the-settings-app-not-opening-in-windows-10

**Comment by grimreaper1014** [comment]
I am on the latest version. I downloaded the windows image software to make a bootable usb so it always gets the latest version. The version I am running is 2004 19041.388. I ended up just reinstalling Windows 10. I am going to make a system restore point this time and see if it happens again.

**Comment by Limeay** [comment]
> I am on the latest version. I downloaded the windows image software to make a bootable usb so it always gets the latest version. The version I am running is 2004 19041.388. I ended up just reinstalling Windows 10. I am going to make a system restore point this time and see if it happens again.

I recommend you just wait till V2009/2010 comes out and it'll fix that issue. Typically new version upgrade fix these issues because it replaces the old Windows folder... I've seen this happen couple times but to wait couple months... :\
Also hoping Sycnex fixes whatever that is to stop deleting some of the core settings as it breaks windows.

**Comment by grimreaper1014** [comment]
I reinstalled Windows 10 and ran the debloat app again. Everything seams to have worked correctly this time. I wonder if it is because I upgraded a very old release last time and ran it? The computer was on 1803 and I updated it all the way to 2004. Maybe there was something that broke because of all them updates. Where as this time I installed 2004 completely updated. I did notice that the script does corrupt some files, but it does not seam to be a big deal as system file checker fixes them.

**Comment by Limeay** [comment]
If only that were true for my system. Unfortunately I'm on 2004 and my desktop windows settings for system does not appear so I cant get to my icc profile settings, hardware gpu scheduling or vrr. I've tried sfc and it doesn't nothing for me. Still broken. I also don't have time to reformat. I'll just wait till 2009/2010 arrives and upgrade and hopefully that'll fix this. Scared to do this for my notebook because it too is on v2004

**Comment by Dan-Burgess** [comment]
I posted this as a reply in another issue but I had this happen twice on in-use installs of Windows.  Debloating fresh installs works as expected.

This PS script fixed my issue:
Get-appxpackage -all *shellexperience* -packagetype bundle |% {add-appxpackage -register -disabledevelopmentmode ($_.installlocation + “\appxmetadata\appxbundlemanifest.xml”)} 

Hope this helps.

**Comment by Limeay** [comment]
> I posted this as a reply in another issue but I had this happen twice on in-use installs of Windows. Debloating fresh installs works as expected.
> 
> This PS script fixed my issue:
> Get-appxpackage -all _shellexperience_ -packagetype bundle |% {add-appxpackage -register -disabledevelopmentmode ($_.installlocation + “\appxmetadata\appxbundlemanifest.xml”)}
> 
> Hope this helps.

Doesn't work. My Settings is still not usable.

**Comment by Limeay** [comment]
Doesn't work. My "System" portion of Settings is still not usable. Though i do notice my Windows has a bit more performance. 
Dan-Burgess, can you tell me where you found your source for this PS script that fixed your issue?
I can't find this on Google.

**Comment by Limeay** [comment]
For anyone that want to have their windows 10 immersivecontrolpanel fixed, the solution is either to reformat your OS or to upgrade to the latest Windows 10 version, for ex: this issue messed up my OS in August, Right now it's October 2020, the newest version just came out. Updated to it and now i don't have this issue anymore and my pc isn't so laggy after it broke my OS.
